frontend/controllers/SiteController.php

```diff
@@ -65,11 +65,21 @@ public function actions()
         ];
     }
 
+    /**
+     * Displays homepage.
+     *
+     * @return mixed
```

Comment:
Without that method phpDoc is incomplete. `@return` can be ommited **only** in case of `void`. If we remove `@return` than PhpStorm IDE will be very angree and will show messages from it strict inspectors. And this is right ;-)